### PR TITLE
RELATED: RAIL-3709 integrate with Productboard

### PR DESCRIPTION
**Pull request by no23reason**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/863/ : SUCCESS in 1m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/565/ : SUCCESS in 8s

File: docs/01_intro__roadmap.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Roadmap
+sidebar_label: Roadmap
+copyright: (C) 2007-2021 GoodData Corporation
+id: roadmap
+---
+
+Below is an interactive view into the current GoodData.UI Roadmap where you can check out the features that are coming to GoodData.UI, comment on them and even suggest new features.
```
Comment (andriichumak) [inline docs/01_intro__roadmap.md:8]: Do we really need any explanation here? To me it seems self explanatory. Unless there are some issues with the layout if we drop the text.

Comment (no23reason) [reply docs/01_intro__roadmap.md]: Good point, removed.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/865/ : SUCCESS in 2m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/567/ : SUCCESS in 7s

File: docs/01_intro__about_gooddataui.md [hunk 2]
```diff
@@ -48,3 +48,7 @@ Because Internet Explorer 11 does not support [CSS variables](https://developer.
 ## Get GoodData
 
 To use GoodData.UI, you must have an account on the GoodData platform or an installation of GoodData.CN. You can create a GoodData platform account for free at [https://www.gooddata.com/free](https://www.gooddata.com/free) or download GoodData.CN at [https://www.gooddata.com/developers](https://www.gooddata.com/developers).
+
+## Upcoming features
+
+Take a look on our [Roadmap](01_intro__roadmap.md) to see what new features are coming to GoodData.UI.
```
Comment (evgeniabogranskaya) [inline docs/01_intro__about_gooddataui.md:54]: Take a look at our [roadmap](01_intro__roadmap.md) to see what new features are coming to GoodData.UI.

File: docs/whats_new_8.md [hunk 3]
```diff
@@ -114,37 +114,41 @@ backend
...
-GoodData backend. You can now use the `@gooddata/mock-handling` tool to simplify creation and management of recorded or 
-mock data. This recorded data can be made available to your application almost transparently, by using a recorded 
+We have created tools and infrastructure to simplify testing of your applications offline, without the real
+GoodData backend. You can now use the `@gooddata/mock-handling` tool to simplify creation and management of recorded or
+mock data. This recorded data can be made available to your application almost transparently, by using a recorded
 backend implementation included in `@gooddata/sdk-backend-mockingbird`.
 
 We are successfully using this in our own testing project `@gooddata/reference-workspace` - check it out.
+
+## Upcoming features
```
Comment (evgeniabogranskaya) [inline docs/whats_new_8.md:152]: Upcoming Features

Comment (evgeniabogranskaya) [inline docs/whats_new_8.md:152]: Upcoming Features

File: docs/whats_new_8.md [hunk 4]
```diff
@@ -114,37 +114,41 @@ backend
...
+
+Take a look on our [Roadmap](01_intro__roadmap.md) to see what new features are coming to GoodData.UI.
```
Comment (evgeniabogranskaya) [inline docs/whats_new_8.md:154]: Take a look at our [roadmap](01_intro__roadmap.md) to see what new features are coming to GoodData.UI.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/870/ : SUCCESS in 1m 30s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/573/ : SUCCESS in 7s

**Review by plavemar — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR460

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/871/ : SUCCESS in 1m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/165/ : SUCCESS in 1m 49s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/60/ : SUCCESS in 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/255/ : SUCCESS in 17s

**Pull request merged**